test/test_StatisticalInverseProblem/test_parallel_h5.sh.in

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -eu
+set -o pipefail
+
+have_mpi=@HAVE_MPI@
+have_hdf5=@HAVE_HDF5@
+
+if [ $have_mpi -eq 1 ]; then
+  if [ $have_hdf5 -eq 1 ]; then
+    PROG="mpirun -np 2 ./test_parallel_h5"
```

Comment:
Maybe use mpiexec here?  I'm not sure which is more common but IIRC mpiexec is the only one that appears (albeit as optional to support...) in an actual MPI standard.